testing/tests/api/helping.py

```diff
@@ -1,50 +0,0 @@
-import sys
```

Comment:
Was this file intended to be removed here?

Reply:
Yes. Is it ok?